scenes/eternal_loom_sokoban/levels/sokoban_template.tscn

```diff
@@ -0,0 +1,56 @@
+[gd_scene load_steps=6 format=4 uid="uid://byvwl6rx7pyps"]
```

Comment:
What's the difference between `sokoban_template`, `eternal_loom_template`, and `eternal_loom_sokoban`?

Reply:
So currently there are two different rulesets, one for "classic sokoban" where it's just about getting every target filled with a crate (in our case clump). And then the eternal loom ruleset requires three different targets to be filled up. So they have two different templates in case someone wants to make a new level for the eternal loom, or just make a classic sokoban puzzle.

`eternal_loom_sokoban` was just the file that was linked to the main game, so I left that alone. That would either be where we put the "randomly choose level" feature, or we put that feature in fray's end on the eternal loom exterior scene and get rid of `eternal_loom_sokoban`.

Reply:
FWIW it's not currently linked to the main game - I unlinked it because I wanted to test the case of entering a quest after completing a quest.

A to-be-answered question is, can a quest specify a sokoban puzzle (or even provide its own), or does the game specify one (or a random one)?

Personally I would put the code to pick a random puzzle into the loom's script.